### ISSUE TITLE
when crop a very large file, R quit without any message.

### ISSUE THREAD

**Issue by jszhao** [rspatial/terra#472]
I have a very large `tif` file (~200GB). I hope to crop it to keep its extent as the same of others. 
```
r <- rast("Rfactor.tif")
crop(r, c(xmin, xmax, ymin, ymax), filename = "Rfactor_m.tif")
```
After a while, R quit without any message, and the output file `Rfactor_m.tif` has very small size.

I run the code in a Win 10 64-bit machine, with RAM 512GB.

**Comment by rhijmans** [comment]
Can you try it again after setting

`terraOptions(memfrac=0.1)`

**Comment by jszhao** [comment]
It works. Thanks!

**Comment by rhijmans** [comment]
To perhaps understand why this was necessary, could you show what you get when you start a fresh R session and do:

```
sessionInfo()$running
library(terra)
terraOptions()
r <- rast("Rfactor.tif")
r
mem_info(r)
```

**Comment by rhijmans** [comment]
To perhaps understand why this was necessary, could you show what you get when you start a fresh R session and do:

```
sessionInfo()$running
library(terra)
terraOptions()
r <- rast("Rfactor.tif")
r
mem_info(r)
```

**Comment by jszhao** [comment]
Here is the information:
```
> sessionInfo()$running
[1] "Windows 10 x64 (build 19042)"
> library(terra)
terra version 1.4.22
> terraOptions()
memfrac   : 0.6
tempdir   : C:/Users/Admin/AppData/Local/Temp/Rtmpq4XZDs
datatype  : FLT4S
progress  : 3
todisk    : FALSE
verbose   : FALSE
tolerance : 0.1
> r <- rast("Rfactor_m.tif")
> r
class       : SpatRaster 
dimensions  : 180985, 295986, 1  (nrow, ncol, nlyr)
resolution  : 30, 30  (x, y)
extent      : -4288665, 4590915, 1608225, 7037775  (xmin, xmax, ymin, ymax)
coord. ref. : WGS 84 / UTM zone 48N (EPSG:32648) 
source      : Rfactor_m.tif 
name        : Rfactor_m 
min value   :         0 
max value   :  25300.63 
> mem_info(r)

------------------------
Memory (GB) 
------------------------
available       : 490.98
allowed (60%)   : 294.59
needed (n=1)    : 399.12
------------------------
proc in memory  : FALSE
nr chunks       : 2
------------------------
> 
```
